[2024-05-20T03:42:31.827+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: temp_humi_dag_csv.delete_data manual__2024-05-20T03:42:30.996419+00:00 [queued]>
[2024-05-20T03:42:31.833+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: temp_humi_dag_csv.delete_data manual__2024-05-20T03:42:30.996419+00:00 [queued]>
[2024-05-20T03:42:31.834+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-20T03:42:31.843+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): delete_data> on 2024-05-20 03:42:30.996419+00:00
[2024-05-20T03:42:31.848+0000] {standard_task_runner.py:60} INFO - Started process 1406 to run task
[2024-05-20T03:42:31.851+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'temp_humi_dag_csv', 'delete_data', 'manual__2024-05-20T03:42:30.996419+00:00', '--job-id', '674', '--raw', '--subdir', 'DAGS_FOLDER/ICTTemp_Humi_CSV.py', '--cfg-path', '/tmp/tmphu9812ts']
[2024-05-20T03:42:31.858+0000] {standard_task_runner.py:88} INFO - Job 674: Subtask delete_data
[2024-05-20T03:42:31.921+0000] {task_command.py:423} INFO - Running <TaskInstance: temp_humi_dag_csv.delete_data manual__2024-05-20T03:42:30.996419+00:00 [running]> on host e3e2dc6bc2be
[2024-05-20T03:42:32.134+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Group4' AIRFLOW_CTX_DAG_ID='temp_humi_dag_csv' AIRFLOW_CTX_TASK_ID='delete_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T03:42:30.996419+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T03:42:30.996419+00:00'
[2024-05-20T03:42:32.140+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T03:42:32.154+0000] {sql.py:450} INFO - Running statement: DELETE FROM test_temp_and_humi;, parameters: None
[2024-05-20T03:42:32.157+0000] {sql.py:459} INFO - Rows affected: 0
[2024-05-20T03:42:32.159+0000] {logging_mixin.py:188} INFO - Delete table successfully from PostgreSQL.
[2024-05-20T03:42:32.161+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-20T03:42:32.167+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=temp_humi_dag_csv, task_id=delete_data, execution_date=20240520T034230, start_date=20240520T034231, end_date=20240520T034232
[2024-05-20T03:42:32.190+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-20T03:42:32.205+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
